packages/common/src/account/sign.ts

```diff
@@ -0,0 +1,103 @@
+// @ts-expect-error types
```

Comment:
what is this for?

Reply:
`asn1.js` doesn't have type definitions. I'm never sure the best way to handle this - should i add a declaration file?

Reply:
can we restrict this to specifically that package? or just ignore the lines that package uses? just worried about this being an overly wide "ignore these errors"

Reply:
I wouldn't block on adding type declarations for this lib though!